src/PowerShell/Help/Microsoft.WinGet.Client/Get-WinGetPackage.md

```diff
@@ -0,0 +1,228 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+ms.date: 08/01/2024
+online version:
+schema: 2.0.0
+title: Get-WinGetPackage
+---
+
+# Get-WinGetPackage
+
+## SYNOPSIS
+Lists installed packages.
+
+## SYNTAX
+
+```
+Get-WinGetPackage [-Tag <String>] [-Command <String>] [-Count <UInt32>] [-Id <String>] [-Name <String>]
+ [-Moniker <String>] [-Source <String>] [[-Query] <String[]>] [-MatchOption <PSPackageFieldMatchOption>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This command lists all of the packages installed on your system. The output includes packages
+installed from WinGet sources and packages installed by other methods. Packages that have package
+identifiers starting with `MSIX` or `ARP` were not installed from a WinGet source.
```

Comment:
Nit: I think there are cases where a package installed from a WinGet source may not be correlated back to it

Reply:
I didn't address this as I'm not quite sure how to best explain this.. 😓